src/tools/debugger/stepping/Stepping.java

```diff
@@ -0,0 +1,42 @@
+package tools.debugger.stepping;
+
+import tools.SourceCoordinate.FullSourceCoordinate;
+/**
+ *
+ * Manage the actor message stepping.
+ *
+ */
+public class Stepping {
+
+  public enum SteppingType{
+    STEP_INTO,
+    STEP_OVER,
+    STEP_RETURN
+  }
+
+  private StepActorOperation stepActor;
+
+  public Stepping(final StepActorOperation stepActor) {
+    this.stepActor = stepActor;
+  }
+  /**
+   * Return the stepping operation that has been made for a specific source section,
```

Comment:
This sounds wrong. How do you know that the next statement is on the next line? it could as well be a line later, or on the same line, no?